src/pylexibank/__init__.py

```diff
@@ -1,5 +1,8 @@
 #
 from pylexibank.dataset import *  # noqa: F401, F403
 from pylexibank.models import *  # noqa: F401, F403
+from pylexibank.forms import *  # noqa: F401, F403
+from pylexibank.metadata import *  # noqa: F401, F403
+from pylexibank.util import *  # noqa: F401, F403
 
```

Comment:
I tend to agree with F401 and F403 in disliking wildcard imports. As a suggestion to the other lexibankers, what do you think of importing just the modules and, in the code, using something like:

```python3
import pylexibank as lb

class my_dataset(lb.dataset.Dataset):
    ...
```

or, perhaps, loading stuff here one by one (like `from pylexibank.dataset import Dataset`, even though it can get long) and then using the code as

```python
import pylexibank as lb

class my_dataset(lb.Dataset):
```

Reply:
I think wildcard imports are ok - in particular from the **same** package - when the imported module has an `__all__` attribute.